include/sdf/Root.hh

```diff
@@ -58,6 +58,14 @@ namespace sdf
     /// \brief Default constructor
     public: Root();
 
+    /// \brief Get the name of the world without loading the entire world
+    /// \param[in] _filename Name of the SDF file to parse.
+    /// \param[out] _worldName The name of the world
+    /// \return Errors, which is a vector of Error objects. Each Error includes
+    /// an error code and message. An empty vector indicates no error.
+    public: Errors WorldName(const std::string &_filename,
```

Comment:
I think this is from a different PR :)

Reply:
https://github.com/gazebosim/sdformat/pull/997/commits/48698b0d1b7e87b165fdb2bc6e0dbca04d4cba75